font.go

```diff
@@ -70,7 +70,25 @@ func (f *Font) Raw() (string, []byte) {
 	return f.mimetype, f.raw
 }
 
-func (f *Font) pdfInfo() (Rect, float64, float64, float64, float64, []int) {
+// Kerning returns the horizontal adjustment for the rune pair. A positive kern means to move the glyphs further apart.
+func (f *Font) Kerning(left, right rune, ppem float64) (float64, error) {
+	var sfntBuffer sfnt.Buffer
+
+	iLeft, err := f.sfnt.GlyphIndex(&sfntBuffer, left)
+	if err != nil {
+		return 0, err
+	}
+	iRight, err := f.sfnt.GlyphIndex(&sfntBuffer, right)
+	if err != nil {
+		return 0, err
+	}
+
+	kern, err := f.sfnt.Kern(&sfntBuffer, iLeft, iRight, toI26_6(ppem), font.HintingNone)
+
+	return fromI26_6(kern), err
+}
+
+func (f *Font) PdfInfo() (Rect, float64, float64, float64, float64, []int) {
```

Comment:
> ultimately this is a bit of an ugly function

It could easily be split into:
- `Bounds`
- `ItalicAngle`
- `Metrics`
- `Widths`

Do you think that it would be better?

Reply:
Maybe as data members (not member functions)? Or just keep it like it is, in the future I want to work on my own SFNT code (which is surprisingly "easy") in order to support more file formats and extract more information from the fonts (some stuff is private for the SFNT package). So depending on how that turns out it may get changed in the future anyways.

Reply:
Actually, might take a while before I'd have time to do that (maybe never? hehe), so let's do it right. Your idea sounds good, some can be functions some can be exported data members I guess? Up to you ;-)

Reply:
I split it into:
- `Bounds`
- `ItalicAngle`
- `Metrics` (I merged it with the Metrics of FontFace)
- `Widths`

> in the future I want to work on my own SFNT code

So this is a good idea, to hide the `sfnt` dependecy from the API :-)